cmd/resultscollector/main.go

```diff
@@ -245,6 +255,39 @@ func main() {
 				fmt.Println(err)
 				os.Exit(1)
 			}
+			fmt.Println("Uploaded ConfigMap")
+
+			err = backoff.Retry(func() error {
+				url := scapresultsconf.ResultServerURI
+				fmt.Printf("Trying to upload to resultserver: %s\n", url)
+				reader := bytes.NewReader(contents)
+				client := &http.Client{}
+				req, err := http.NewRequest("POST", url, reader)
```

Comment:
Does it make sense to check the error here?

Reply:
uhm... perhaps not. The error would only be returned if we gave the wrong parameters.

Reply:
Fixed.